profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -101,7 +112,12 @@ function _tmgmt_poetry_authenticate($user, $password) {
  * Function available to call from our webservice.
  */
 function FPFISPoetryIntegrationRequest($user, $password, $msg) {
-  watchdog('Poetry: Response', filter_xss(htmlentities($msg)), array(), WATCHDOG_DEBUG);
+  watchdog(
+    'tmgmt_poetry',
+    "Receive request: !msg",
+    ['!msg' => htmlentities($msg)],
```

Comment:
Previously, we use "filter_xss(htmlentities($msg))" and I think that it is a double verification of $msg.